Doc/using/configure.rst

```diff
@@ -289,6 +289,143 @@ General Options
 
    .. versionadded:: 3.13
 
+.. cmdoption:: PKG_CONFIG
+
+   Path to ``pkg-config`` utility.
+
+.. cmdoption:: PKG_CONFIG_LIBDIR
+.. cmdoption:: PKG_CONFIG_PATH
+
+   ``pkg-config`` options.
+
+
+C Compiler Options
+------------------
+
+.. cmdoption:: CC
+
+   C compiler command.
+
+.. cmdoption:: CFLAGS
+
+   C compiler flags.
+
+.. cmdoption:: CPP
+
+   C preprocessor command.
+
+.. cmdoption:: CPPFLAGS
+
+   C preprocessor flags, e.g. ``-I<include dir>``.
+
+
+Linker Options
```

Comment:
Nitpick:

```suggestion
Linker options
```

Reply:
ditto: I prefer to capitalize all titles.